### PR TITLE
[Modular] Adds the Dom Aura quirk to the dominatrix cyborg module, and makes it researchable/printable in the utility upgrades

### PR DESCRIPTION
**Pull request by Odairu**
## About The Pull Request
Does as the title says

## Why It's Good For The Game
Cyborgs deserve to have some pets too, as a treat. Very useful for those who want to ERP with these quirks but the dom is a silicon

## Proof Of Testing
image

## Changelog

:cl:
add: DomAura quirk to dom borg module, and adds it to the tech tree
spellcheck: Untility Cyborg Upgrades -> Utility Cyborg Upgrades
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/quirks/code/negative_quirks/well_trained.dm [hunk 1]
```diff
@@ -22,7 +22,7 @@
 /datum/quirk/well_trained/proc/on_dom_examine(atom/source, mob/living/user, list/examine_list)
 	SIGNAL_HANDLER
 
-	if(!ishuman(user))
+	if(!isliving(user))
 		return
 	var/mob/living/carbon/human/dom = user
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/quirks/code/negative_quirks/well_trained.dm:27]:
```suggestion
	var/mob/living/dom = user
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]

File: modular_zubbers/code/modules/quirks/code/negative_quirks/well_trained.dm (hunk 1 above)
Comment (Majkl-J) [reply modular_zubbers/code/modules/quirks/code/negative_quirks/well_trained.dm:27]: Wait better idea

File: modular_zubbers/code/modules/quirks/code/negative_quirks/well_trained.dm [hunk 2]
```diff
@@ -22,7 +22,7 @@
 /datum/quirk/well_trained/proc/on_dom_examine(atom/source, mob/living/user, list/examine_list)
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/quirks/code/negative_quirks/well_trained.dm:22]:
```suggestion
/datum/quirk/well_trained/proc/on_dom_examine(atom/source, mob/living/dom, list/examine_list)
```

File: modular_zubbers/code/modules/quirks/code/negative_quirks/well_trained.dm [hunk 3]
```diff
@@ -22,7 +22,7 @@
...
 	SIGNAL_HANDLER
 
-	if(!ishuman(user))
+	if(!istype(user))
 		return
 	var/mob/living/carbon/human/dom = user
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/quirks/code/negative_quirks/well_trained.dm:27]:
```suggestion
	if(!istype(dom))
		return
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Review by Majkl-J — APPROVED** [review]

**Review by projectkepler-ru — APPROVED** [review]

**Pull request merged**